src/compiler/printer/unique.ts

```diff
@@ -0,0 +1,65 @@
+import { UniqueIdentifierFlags } from '../ast/node';
+import { UniqueIdentifier } from '../ast/internal/unique-identifier';
```

Comment:
@aladdin-add same with this file. Need to be *deleted*. Not ready yet and doesn't work

Reply:
done.